testproject/Assets/Tests/Runtime/MultiprocessRuntime/ExecuteStepInContextTests.cs

```diff
@@ -0,0 +1,256 @@
+using System;
+using System.Collections;
+using System.Text.RegularExpressions;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using static ExecuteStepInContext;
+
+namespace MLAPI.MultiprocessRuntimeTests
+{
+    /// <summary>
+    /// Smoke tests for ExecuteStepInContext, to make sure it's working properly before being used in other tests
+    /// </summary>
+    [TestFixture(1)]
+    [TestFixture(2)]
+    public class ExecuteStepInContextTests : BaseMultiprocessTests
+    {
+        private int m_NbWorkersToTest;
+        public ExecuteStepInContextTests(int nbWorkersToTest)
```

Comment:
and once again, `NbWorkers` to something more easy to understand? :)

Reply:
you can't say I'm not consistent :P